src/Npgsql.OpenTelemetry/Npgsql.OpenTelemetry.csproj

```diff
@@ -17,6 +17,10 @@
     </ItemGroup>
 
     <ItemGroup>
-        <None Include="README.md" Pack="true" PackagePath="\"/>
+        <None Include="README.md" Pack="true" PackagePath="\" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Compile Remove="..\Shared\CodeAnalysis.cs" />
```

Comment:
Any particular reason for doing this?

Reply:
Once I made internals of the main project visible to this project, it was causing a build error due to having duplicate types visible:
```
Error CS0436 : The type 'DynamicallyAccessedMemberTypes' in '<REDACTED>/npgsql/src/Npgsql.OpenTelemetry/../Shared/CodeAnalysis.cs' conflicts with the imported type 'DynamicallyAccessedMemberTypes' in 'Npgsql, Version=7.0.0.0, Culture=neutral, PublicKeyToken=5d8b90d52f46fda7'. Using the type defined in '<REDACTED>/npgsql/src/Npgsql.OpenTelemetry/../Shared/CodeAnalysis.cs'.
```

Reply:
OK, doesn't matter much. I originally thought of Npgsql.OpenTelemetry as more of optional sugar, so that you could configure stuff without taking a dependency on it - but it's really not very important.

Let's keep things as they are in your PR, we can always revisit later.